src/components/toolbar.tsx

```diff
@@ -118,6 +104,10 @@ export class ToolbarComponent extends BaseComponent<IProps, {}> {
     }
   }
 
+  private handleSelect() {
+    // nothing to do
```

Comment:
Is it intentional that the select tool does nothing?  Why do we have it?

Reply:
Previously it was implicit that the select tool did nothing. The switch statement would call `handleAddToolTile()` which called `DocumentModel.addTile()` which called `DocumentContentModel.addTile()` which ignored it because it's not a valid tile type. I'm just making the fact that it does nothing explicit.

As for _why_ it does nothing, I think it was added at a time when it was assumed that some of the toolbar buttons would be modal, like in a drawing package when you can be in "rectangle" mode or "line" mode, in which case you would need a tool to get back to the "default" or "select" mode. But at the document level, none of the tools are actually modal, so the select button doesn't have anything to do.

That said, none of that is germane to this PR except for the fact that I'm choosing to make the no-op behavior of the select tool explicit rather than implicit.

Reply:
Makes sense.  Thanks for filling me in.